### PR TITLE
add rocketpool description to pool staking services [Fixes #4891]

### LINKED ISSUE #4891: Add Rocketpool to 'Less Than 32 ETH' section of staking page
**Describe the solution you'd like**
The current [staking page](https://ethereum.org/en/eth2/staking/#stake) under the `less than 32 eth` section only links out to a [list of pools and staking services](https://beaconcha.in/stakingServices). Now that Rocketpool is live, I think it would be great to feature that project. As they are more than just a pool, users with less than 32 eth can actually run a node. 

Happy to hear your thoughts on this, I have a PR ready to go if this would be something that you'd be open to adding.

### PR DESCRIPTION
**Pull request by MiConnell**
## Description

This PR adds a brief description of rocket pool to the `less than 32 eth` section of the staking page

## Related Issue

discussed on issue #4891

### CODE REVIEW

File: src/intl/en/page-staking.json [hunk 1]
```diff
@@ -32,10 +32,11 @@
   "page-staking-more-sharding": "More on sharding",
   "page-staking-pool": "Stake in a pool",
   "page-staking-pool-desc": "If you have less than 32 ETH, you will be able to add a smaller stake to staking pools. Some companies can do it all on your behalf so you won't have to worry about staying online. Here are some companies to check out.",
+  "page-staking-rocket-pool": "Rocket Pool allows users with only 16 ETH to run a node on the Ethereum network by pairing the node operator's ETH with 16 more from the protocol's deposit pool. If you'd like to participate in the protocol without running a node, you can swap ETH for their liquid token rETH with as little as 0.01 ETH. This is the only truly decentralized staking pool.",
```
Comment (minimalsm) [inline src/intl/en/page-staking.json:35]:
```suggestion
  "page-staking-rocket-pool": "Using Rocket Pool, you can run a staking node starting from 16 ETH. The remaining ETH gets assigned from the Rocket Pool protocol, using ETH users have deposited to the pool. If you'd like to participate in the protocol without running a node, you can swap ETH for their liquid token rETH with as little as 0.01 ETH. This is the only truly decentralized staking pool.",
```

**Review by minimalsm — CHANGES_REQUESTED** [review]
Thanks for the PR @MiConnell ❤️

Made a small suggestion to change it to the first person perspective and simplify the explanation.

File: src/intl/en/page-staking.json (hunk 1 above)
Comment (samajammin) [reply src/intl/en/page-staking.json:35]: I feel like mentioning staking as a user (vs a node operator) should be the first thing to mention.

**Comment by minimalsm** [comment]
Added @samajammin's suggestion and merging. 

Thanks again for the PR @MiConnell. Be sure to claim your [claim your 2022 POAP](https://ethereum.org/en/contributing/#poap) 💪

**Review by minimalsm — APPROVED** [review]

**Pull request merged**